src/EnergyPlus/Psychrometrics.cc

```diff
@@ -449,7 +400,7 @@ namespace Psychrometrics {
 
         // FUNCTION PARAMETER DEFINITIONS:
         int constexpr itmax(100); // Maximum No of Iterations
-        static constexpr std::string_view RoutineName("PsyTwbFnTdbWPb");
+        static constexpr std::string_view RoutineName(PsyRoutineNames[static_cast<int>(PsychrometricFunction::RhFnTdbRhov)]); // PsyTwbFnTdbWPb
```

Comment:
Keep the `static` keyword? I left `RoutineName`s as separate variables for readability.